tests/test_mlip_calculators.py

```diff
@@ -102,10 +106,13 @@ def test_invalid_device(architecture):
     "architecture, device, kwargs",
     [
         ("alignn", "cpu", {}),
-        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024"}),
-        ("alignn", "cpu", {"model_path": MODEL_PATH / "v5.27.2024/best_model.pt"}),
+        ("alignn", "cpu", {"model_path": ALIGNN_PATH}),
+        ("alignn", "cpu", {"model_path": ALIGNN_PATH / "best_model.pt"}),
         ("alignn", "cpu", {"model": "alignnff_wt10"}),
-        ("alignn", "cpu", {"path": MODEL_PATH / "v5.27.2024"}),
+        ("alignn", "cpu", {"path": ALIGNN_PATH} ),
+        ("sevennet", "cpu", {"model": SEVENNET_PATH}),
+        ("sevennet", "cpu", {"path": SEVENNET_PATH}),
+        ("sevennet", "cpu", {"model_path": SEVENNET_PATH}),
```

Comment:
We probably should test the string labels too (https://github.com/MDIL-SNU/SevenNet/tree/main?tab=readme-ov-file#training) - one of `"base"`, `"fine_tune"`, and `"sevennet-0"`? I think these should work as is.

Reply:
Did you add a test for this?

Now that there's a default, I think we want to test that it works both with `kwargs = {"model_path": "sevennet-0"}` and `kwargs={}` (which should hopefully do the same).

Reply:
done now